app/Models/Vote.php

```
@@ -0,0 +1,228 @@
+                    'weight' => $this->apartmentVotes()->where('choice', 'no')->sum('weight'),
+                    'percentage' => 0,
+                ],
+                'abstain' => [
+                    'count' => $this->apartmentVotes()->where('choice', 'abstain')->count(),
+                    'weight' => $this->apartmentVotes()->where('choice', 'abstain')->sum('weight'),
+                    'percentage' => 0,
+                ],
+            ];
+
+            if ($totalWeight > 0) {
+                foreach ($results as $choice => &$data) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$choice'. (Unused Code Rules)

(UnusedLocalVariable)